sdk/core/Azure.Core/src/Shared/RetryAfterDelayStrategy.cs

```diff
@@ -34,9 +34,10 @@ public RetryAfterDelayStrategy(DelayStrategy? fallbackStrategy = null)
         /// If retry-after header is not found, adopt the value of last retry-after header.
         /// </summary>
         /// <param name="response">Service response which might carry retry-after header.</param>
+        /// <param name="attempt"></param>
```

Comment:
Would the shared implementations move into the public API as well?  Or would they be sealed and made internal and moved out of Shared?

Reply:
They would still need to be shared since they are still in other libraries.

Reply:
What would our longer-term strategy be for them?

Reply:
I've removed the shared strategies for exponential and constant and replaced with public strategies.